pages/design-tokens/spacing-units.md

```diff
@@ -56,6 +56,8 @@ Any spacing value in utilities or component CSS should use the following spacing
             <span class="text-no-wrap padding-right-4">
               {% if converted %}
                 <code>{{ converted }}</code>,
+              {% elsif item.conversion %}
```

Comment:
I would just suggest we add a note for what this means. That way we can understand the logic flow in the future.

Reply:
Added comments in [b4b7ad7](https://github.com/uswds/uswds-site/pull/1949/commits/b4b7ad7227d2f89167ba90e951cb5dcec3990b00)